core/services/ocr2/plugins/ocr2keeper/evm/registry.go

```diff
@@ -2,30 +2,44 @@ package evm
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"math/big"
+	"net/http"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	coreTypes "github.com/ethereum/go-ethereum/core/types"
+
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/patrickmn/go-cache"
```

Comment:
is this a vetted library?

Reply:
what is the process of vetting it? 

@cmalec did you ask security to vet it?

Reply:
https://smartcontract-it.atlassian.net/servicedesk/customer/portal/4/SECHD-6620